splinepy/microstructure/tiles/hollow_octagon.py

```
@@ -51,262 +58,418 @@ def _closing_tile(
+
+                bottom_right = _np.array(
+                    [
+                        [v_inner_c_h + v_one_half, -v_h_void + v_one_half],
+                        [v_outer_c_h + v_one_half, v_zero],
+                        [v_h_void + v_one_half, -v_inner_c_h + v_one_half],
+                        [v_one, -v_outer_c_h + v_one_half],
+                    ]
+                )
+
+            for control_points in [
+                right,
```

Pylint (3.3.7)

[error] Possibly using variable 'right' before assignment

(E0606)